third_party/py/python_configure.bzl

```diff
@@ -155,8 +155,8 @@ def _get_python_include(repository_ctx, python_bin):
             python_bin,
             "-c",
             "from __future__ import print_function;" +
-            "from distutils import sysconfig;" +
-            "print(sysconfig.get_python_inc())",
+            "import sysconfig;" +
+            "print(sysconfig.get_path('include'))",
```

Comment:
Will this work on Py3.7-Py3.9?

We need to have the exact same code be used for all supported Python versions.

Reply:
Hi, I've tested on Python 3.6 - 3.10 before created the pull request. 

It should work because `sysconfig` was introduced [since Python 3.2](https://docs.python.org/3.10/library/sysconfig.html#:~:text=Python%E2%80%99s%20configuration%20information-,New%20in%20version%203.2.,-Source%20code%3A%20Lib).

Reply:
Awesome. Thank you for confirming and for the PR